src/data-model-providers/codegen/CodegenDataModelProvider.h

```diff
@@ -187,6 +187,9 @@ class CodegenDataModelProvider : public DataModel::Provider
 
     void Temporary_ReportAttributeChanged(const AttributePathParams & path) override;
 
+protected:
+    virtual void InitDataModel();
```

Comment:
Again: this should be documented/named to make it clear that this is a temporary hack for a test, and that nothing else should rely on it.